src/SFA.DAS.Authorization.WebApi/AuthorizationFilter.cs

```diff
@@ -0,0 +1,30 @@
+﻿#if NET462
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http.Controllers;
+using System.Web.Http.Filters;
+using WebApi.StructureMap;
+
+namespace SFA.DAS.Authorization.WebApi
+{
+    public class AuthorizationFilter : ActionFilterAttribute
+    {
+        public override void OnActionExecuting(HttpActionContext actionContext)
+        {
+            var dasAuthorizeAttributes = actionContext.ActionDescriptor.GetDasAuthorizeAttributes().ToList();
```

Comment:
I'd be tempted to remove the `ToList()` (even though it causes Resharper to complain about possible multiple enumerations).
If there aren't any attributes, the `Any()` call will return as soon as it finds the enumerable empty, and there was no value in creating a list.
If there are attributes, the 'Any()` call should return when it finds the first attribute, and there is only 1 full enumeration during the `SelectMany()` call, so again there doesn't seem to be value in creating a List.